platform-emf/src/test/java/com/softicar/platform/emf/attribute/field/foreign/entity/input/EmfEntityInputTest.java

```diff
@@ -118,6 +118,19 @@ public void testEditButtonWithValidEntityButWithoutRight() {
 		assertTrue(findEditButton().isNodeDisabled());
 	}
 
+	@Test
+	public void testEditEntityWithEditButton() {
+
+		object23.addAuthorizedUser(user);
+		openBrowsePopover();
+		findNode(DomPopover.class).clickNode(IDisplayString.create("twentythree"));
+		clickEditButton();
+		var formPopup = findFormPopup(EmfTestObject.class);
+		formPopup.setInputValue(EmfTestObject.NAME, "edited");
+		formPopup.clickSaveAndCloseButton();
+		assertSame(object23, getValue());
```

Comment:
here, we never check the important fact, that the text in the input actually changed